src/ol/worker/webgl.js

```diff
@@ -29,7 +29,7 @@ worker.onmessage = (event) => {
     const indexBuffer = new Uint32Array(indicesCount);
     const vertexBuffer = new Float32Array(verticesCount);
 
-    let bufferPositions = null;
+    let bufferPositions = undefined;
```

Comment:
```suggestion
    let bufferPositions;
```

Reply:
Ok. I did this because of a `Variable might not have been initialized ` warning, but I see it's just a webstorm warning, not TS or linter.